anta/tests/cvx.py

```diff
@@ -87,3 +87,75 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyMcsServerMounts(AntaTest):
+    """Verify if all MCS server mounts are in mountStateMountComplete.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the MCS mount status on MCS server are mountStateMountComplete.
+    * Failure: The test will fail even if one switch's MCS server mount status is not  mountStateMountComplete.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+    - VerifyMcsServerMounts:
+        expected_connection_count: 100
+    ```
+    """
+
+    categories: ClassVar[list[str]] = ["cvx"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show cvx mounts", revision=1)]
+
+    class Input(AntaTest.Input):
+        """Input model for the VerifyMcsServerMounts test."""
+
+        expected_connection_count: int
+        """The expected number of active CVX Connections with mountStateMountComplete"""
+
+    def validate_mount_states(self, mount: dict[str, Any], mcs_path_types: list[str]) -> None:
+        """Validate the mount states of a given mount."""
+        mount_states = mount["mountStates"][0]
+        num_path_states = len(mount_states["pathStates"])
+        if num_path_states != len(mcs_path_types):
+            self.result.is_failure(f"Unexpected number of mount path states: {num_path_states}")
+
+        for path in mount_states["pathStates"]:
+            path_type = path["type"]
+            path_state = path["state"]
+            if path_type not in mcs_path_types:
+                self.result.is_failure(f"Unexpected MCS path type: {path_type}")
+            if path_state != "mountStateMountComplete":
+                self.result.is_failure(f"MCS server mount state is not valid: {path_state}")
+
+    @AntaTest.anta_test
+    def test(self) -> None:
+        """Main test function for VerifyMcsServerMounts."""
+        command_output = self.instance_commands[0].json_output
+        self.result.is_success()
+        connections = command_output["connections"]
+        active_count = 0
+        mcs_path_types = ["Mcs::ApiConfigRedundancyStatus", "Mcs::ActiveFlows", "Mcs::Client::Status"]
```

Comment:
are we sure about this list to be hardcoded? seems like it is used to validate the length later - should it be a param? otherwise let's set it as a class variable so you don't need to pass it to the child function `validate_mount_states` as an argument